grunttasks/l10n-htmllint.js

```diff
@@ -34,10 +34,10 @@ module.exports = function (grunt) {
         }
       }
       if (content !== '') {
-        grunt.file.write(path.join('<%= yeoman.tmp %>', pathname[1], pathname[2], pathname[3].replace('json', 'html')), content);
+        grunt.file.write(path.join('.tmp', pathname[1], pathname[2], pathname[3].replace('json', 'html')), content);
```

Comment:
Not exactly broken, but it created a new folder called <%= yeoman.tmp %>, which was then checked by the parser, but this is better, really.